scripts/alphabetizeTypes.js

```diff
@@ -0,0 +1,33 @@
+const Comments = require('parse-comments');
+const TypescriptParser = require('typescript-parser').TypescriptParser
+const argv = require('yargs').argv
+const readFileSync = require('fs').readFileSync;
+const writeFileSync = require('fs').writeFileSync;
+const comments = new Comments();
+const parser = new TypescriptParser();
+
+const FILENAME = argv.i;
+const HEADER = argv.h;
+
+(async () => {
+    const original = readFileSync(FILENAME).toString();
+    const parsedCode = await parser.parseFile(FILENAME).catch(err => {
+        console.error(err);
```

Comment:
Could you wrap the entire function with a catch instead?

Reply:
yes